README.md

```diff
@@ -1,8 +1,131 @@
+
+ Researched and provided proof of concepts and illustrations on providing predictive data models as a service to end users, as well as debugging / redesigning old VBA codes and assistance with JS implementation in Cognos. 
+
+
+**Software Development – COOP, Symcor Inc. (MAY2019-DEC2019)**: 
+
+ Assist in qualitative research to gain an understanding of the human needs involved in designing solutions and to help reframe the problem in human-centric ways through the design thinking process. Research work also included trends in solutions and competitive landscape. 
+
+ Distill information from the qualitative research to create design thinking personas and to generate multiple solution ideas for that persona. Adopted a hands-on approach in prototyping those ideas; including sketching and wireframes. 
+
+ Maintained and updated internal website built on PHP. Mostly front-end work. 
+
+
+**Predictive Modelling Analyst – COOP, Rogers Communication (SEP2018-DEC2018)**: 
+
+ Accessed information in the databases to create data extracts and cleansing, transformations and summaries by using Proc SQL scripting within SAS Enterprise, and provided these for marketing initiatives. 
+
+ Visualized data through excel dashboards by means of pivot tables and charts which are then used in PowerPoint presentations to internal / external teams within the marketing and sales business unit. 
+
+ Reviewed data to look at patterns and unusual data, and perform sample testing and/or research to validate it. 
+
+ Constructed supervised and unsupervised churn models and ensure their end-to-end performance validation. 
+
+
+
+## Education
+**Ryerson University (SEP2015-DEC2020)**, BSc (Honours), Computer Science. Software Engineering concentration, Computer Science
+- Honours Degree
+- Consistent Dean's List
+- GPA: 3.72/4.30
+
+**CGA Canada (SEP2009-DEC2014)**, CPA, CGA Certification, Accounting
+- Professional Designation
+
+**SAIT Polytechnic (SEP2010-DEC2013)**, Bachelor Applied Business Administration Accounting and IT, Business Administration
+- Honours Degree
+- GPA: 3.62/4.00
+
+
+## Miscellaneous
+- SOFT SKILLS:
+- Strength in problem solving, design and analysis and a quick learner
+- Have a strong desire to continually develop and grow both personally and professionally
+- Consistently recognized as being an effective worker who thrives on new challenges and responsibilities
+- Possesses excellent interpersonal and organizational skills
+- Adapts a customer-centric approach to work ethic for both internal and external stakeholders
+- Known for going above and beyond the job description and assisting co-workers / team members
+- Highly intuitive and applies the same principle to work produced
+- Takes initiative in acquiring necessary resources independently in ensuring task completion
+- Works effectively well independently or collaboratively with others
+- :
+- ACTIVITIES/HOBBIES:
+- Gardening
+- Volunteer, Carnegie Community Centre:  Kitchen Help
```

Comment:
remove soft skills. you should talk about those things throughout the profile (e.g. in the experience or project sections).

Reply:
let's use the king of format for this section

```
others:
    #category
  - title: "Achievements"
    #list of items
    bullets:
      - "Selected to showcase projects to Tim Cook (Apple CEO) - Jan 23, 2018"
      - "Selected to showcase projects to Honourable Deb Matthews (Minister of Advanced Education and Skills Development) - Jan 17, 2017"
  - title: "Awards"
    bullets:
      - "Mobile Computer Symposium Competition - First Place (Oct 25, 2016)"
  - title: "Activities/Hobbies"
    bullets:
      - "Archery"
      - "Computer Building/Modifications"
      - "Building DIY Tech Devices"
```